### PR TITLE
Added GRCm38 genome build for 10X data alignment

### PR DESCRIPTION
**Pull request by nilseling**
Dear Aaron Lun,
could you please merge this to the current version of the file?
Thanks heaps!
Nils Eling

### CODE REVIEW

File: genomes/10X/README.md [hunk 1]
```diff
@@ -54,5 +54,17 @@ bsub -R "rusage[mem=32000]" -n 4 -e "xtr.err" -o "xtr.out" \
     --fasta=/scratchb/jmlab/resources/genomes/sequences/B.LAN_REFERENCE/Bl71nemr.fa \
     --genes=/scratchb/jmlab/resources/annotation/processed/B.LAN_exons.gtf
 ```
+## Generating a mus musculus package from GRCm38 genome build
```
Comment (LTLA) [inline genomes/10X/README.md]: Upper case and italics.

**Comment by LTLA** [comment]
ITALICS

**Comment by jcmarioni** [comment]
Surely the response should be: UPPER CASE

Otherwise there’s a logic fail, no?

J

---

John Marioni PhD
Research Group Leader, EMBL-EBI
Associate Faculty Member, WT Sanger Institute
Wellcome Genome Campus
CB10 1SD, UK

Senior Group Leader
CRUK Cambridge Institute
University of Cambridge
CB2 0RE, UK

**Comment by LTLA** [comment]
My mathematician's answer to your question is: yes.

**Comment by jcmarioni** [comment]
Ai - I’m going to regret starting this thread…

---

John Marioni PhD
Research Group Leader, EMBL-EBI
Associate Faculty Member, WT Sanger Institute
Wellcome Genome Campus
CB10 1SD, UK

Senior Group Leader
CRUK Cambridge Institute
University of Cambridge
CB2 0RE, UK

**Pull request merged**